project/backend/api/views.py

```diff
@@ -346,3 +346,39 @@ def get_workspace_from_id(request):
                         'references':references,
                          'created_at':workspace.created_at,
                          }, status=200)
+
+@api_view(['POST'])
+def send_collaboration_request(request):
+
+    serializer = CollaborationRequestSerializer(data=request.data)
+    if serializer.is_valid():
+        serializer.save()
+        return Response(serializer.data, status=201)
+    return Response(serializer.errors, status=400)
+
+@api_view(['PUT'])
+def update_request_status(request):
+    try:
+        req = Request.objects.get(pk=request.data.get('workspace_id'))
```

Comment:
workspace_id?

Reply:
Workspace model uses "workspace_id" as its primary key. Should we access the workspaces in different way?